pkg/apis/servicecatalog/doc.go

```diff
@@ -19,5 +19,5 @@ limitations under the License.
 
 // Package servicecatalog contains all of the sources needed to make
 // servicebrokers and their related service objects.
-// +groupName=
+// +groupName=servicecatalog.k8s.io
```

Comment:
@duglin I think in theory the codegen pulls this from the package name, and there was some restriction on not having dots in the name. I'd still like to know if anyone uses it that way and if we can throw an error or pull this information from somewhere else. I know we have this string written down in ten or twenty places in the code and it should really be all in one place, not all over.